tests/CMakeLists.txt

```diff
@@ -39,7 +39,6 @@ if(GTEST_FOUND)
   if(UNITTEST_STATIC_LINK)
     target_link_libraries(${PROJECT_NAME}_unit_tests
       ${GTEST_LIBRARY}
-      rt
```

Comment:
mxnet already declares rt as a link dep.  The way in which mxnet is more platform independent than this mechanism, so I'd recommend we just let the mxnet targets declare this dep.

Reply:
Declares it into which macro? Is that macro used here?  Is it added into the macronin such a way that it's always part of the macro at this point?

Reply:
I should make it clear that I'm not a cmake expert, but to me it looks like it'll be included in mxnet_LINKER_LIBS from line https://github.com/apache/incubator-mxnet/blob/master/CMakeLists.txt#L225 when required (i.e. in Unix, not apple).  and mxnet_LINKER_LIBS is referenced both here https://github.com/apache/incubator-mxnet/blob/master/tests/CMakeLists.txt#L45 and here https://github.com/apache/incubator-mxnet/blob/master/tests/CMakeLists.txt#L54 in the same scope as rt.